build/commands/lib/config.js

```diff
@@ -771,6 +771,9 @@ Object.defineProperty(Config.prototype, 'defaultOptions', {
     if (this.use_goma && this.gomaServerHost) {
       env.CC_WRAPPER = path.join(this.depotToolsDir, '.cipd_bin', 'gomacc')
       env.GOMA_SERVER_HOST = this.gomaServerHost
+      if (process.platform === 'darwin') {
+        args.use_system_xcode = false
```

Comment:
this is not the right place to put this and it won't even run like this, it will just be an error